src/ResourceManager/Compute/Commands.Compute/Generated/Snapshot/Config/NewAzureRmSnapshotUpdateConfigCommand.cs

```diff
@@ -121,76 +122,11 @@ protected override void ProcessRecord()
 
         private void Run()
         {
-            // CreationData
-            Microsoft.Azure.Management.Compute.Models.CreationData vCreationData = null;
-
             // EncryptionSettings
             Microsoft.Azure.Management.Compute.Models.EncryptionSettings vEncryptionSettings = null;
 
-#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
If these values are provide, it does not affect the output type.  Actually, even a user provides these values to REST body, the server ignores that and does not update it.  So there is no behavior change.  We make this parameter to obsolete not to confuse users.